channel/boss.go

```diff
@@ -32,12 +32,17 @@ func manageSummonedBoss(inst *fieldInstance, mobID int32, server *Server) {
 			}
 
 			inst.dispatch <- func() {
-				for _, mob := range inst.lifePool.mobs {
-					if mob.boss {
-						return
+				if len(inst.players) > 0 {
+					for _, mob := range inst.lifePool.mobs {
+						if mob.boss {
+							return
+						}
 					}
 				}
 
+				inst.lifePool.eraseMobs()
+				inst.dropPool.eraseDrops()
```

Comment:
This will erase drops from mob potentially before player picks them up after killing ths boss?